packages/server/seeds/01_organisation-developers.js

```diff
@@ -12,36 +12,46 @@ exports.seed = async (knex) => {
     console.log(knex.client.config)
 
     await knex('user_organisation').del()
+    await knex('user_external_id').del()
     await knex('users').del()
     await knex('organisation').del()
 
-    console.log('Inserting organisations')
-
-    //Organisations
-    await knex('organisation').insert([
-        { id: 1, uuid: uuid(), name: 'DHIS2', slug: slugify('DHIS2', {lower: true}) },
-        { id: 2, uuid: uuid(), name: 'World Health Organization', slug: slugify('World Health Organization', {lower: true}) },
-    ])
 
     console.log('Inserting users')
 
     //Developers
     await knex('users').insert([
-        { id: 1, uuid: 'd30bfdae-ac6e-4ed4-8b2c-3cd1787922f4', email: 'erik@dhis2.org', first_name: 'Erik', last_name: 'Arenhill' },
-        { id: 2, uuid: '71bced64-c7f7-4b70-aa09-9b8d1e59ed49', email: 'viktor@dhis2.org', first_name: 'Viktor', last_name: 'Varland' },
+        { id: 1, uuid: '58262f57-4f38-45c5-a3c2-9e30ab3ba2da', email: 'appstore-api@dhis2.org', name: 'Mr Jenkins' },
+        { id: 2, uuid: 'd30bfdae-ac6e-4ed4-8b2c-3cd1787922f4', email: 'erik@dhis2.org', name: 'Erik Arenhill' },
+        { id: 3, uuid: '71bced64-c7f7-4b70-aa09-9b8d1e59ed49', email: 'viktor@dhis2.org', name: 'Viktor Varland' },
+    ])
+
+    console.log('Inserting user_external_id')
+    await knex('user_external_id').insert([
+        { id: 1, user_id: 1, external_id: `${process.env.auth0_audience}@clients` }
     ])
 
+    console.log('Inserting organisations')
+
+    //Organisations
+    await knex('organisation').insert([
+        { id: 1, uuid: uuid(), name: 'DHIS2', slug: slugify('DHIS2', { lower: true }), created_by_user_id: 1 },
+        { id: 2, uuid: uuid(), name: 'World Health Organization', slug: slugify('World Health Organization', { lower: true }), created_by_user_id: 1 },
+    ])
+
+
     console.log('Inserting user-organisations #01')
     //user-organisations
     await knex('user_organisation').insert([
-        { organisation_id: 1, user_id: 2 },
-        { organisation_id: 2, user_id: 1 },
+        { organisation_id: 1, user_id: 3 }, //viktor -> dhis2
```

Comment:
i'm finally famous enough to have made it into a test

Reply:
yes, the answer to life is not 42, it's to be included into data for unit tests.